platforms/native/language-native/src/main/java/org/gradle/language/nativeplatform/internal/incremental/IncrementalCompileProcessor.java

```diff
@@ -49,10 +48,9 @@ public IncrementalCompilation call(BuildOperationContext context) {
 
             @Override
             public BuildOperationDescriptor.Builder description() {
-                ProcessSourceFilesDetails operationDetails = new ProcessSourceFilesDetails(sourceFiles.size());
```

Comment:
off topic but still justifiable as just a refactoring allowing the change. Actually did only to have unrelated change wanted to check this initially. 

Let me know if acceptable or undo, thanks.